internal/c/no_cancel_ctx.go

```diff
@@ -0,0 +1,20 @@
+package c
+
+import (
+	"context"
+	"time"
+)
+
+type noCancel struct {
+	ctx context.Context
```

Comment:
not a big deal at all, but why not `s/ctx context.Context/context.Context/`

Reply:
Not sure how to make it work that way :grimacing:

Reply:
I just meant embedding: https://golang.org/doc/effective_go.html#embedding